frontend/src/globals/unity-util.ts

```diff
@@ -1543,7 +1543,7 @@ export class UnityUtil {
 		toggleMode: boolean,
 		forceReHighlight: boolean,
 	) {
-		UnityUtil.multipleCallInChunks(idArr.length, (start, end) => {
+		return UnityUtil.multipleCallInChunks(idArr.length, (start, end) => {
```

Comment:
how does changing the code in highlight fix an issue with colour overrides?

If we do this then shouldn't we do this on everything that is related? (e.g. visibility, colour override, transparency)

Reply:
I think the issue title is a bit wrong. I thought the same thing when Alessandro talked to me about it, but he was refering to the hghlights that are shown when selecting a group, not the the actual mesh override. 

<img width="752" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/d9b05f4d-2fc7-46c1-8898-9eb788731276">

Reply:
but i'm thinking if this is an issue for highlight, woudl it not be an issue for others? (e.g. if you flip through issues/risks/tickets)

should we not be changing the behaviour for all?

It's a bit weird that highglith returns a promise but not override/visibilty etc

Reply:
My guess is that the problem is there but we haven't found a consistent way of reproducing it for the rest. Or maybe the delay when loading is preventing the thing to manifest. Again, these are all guesses.

Reply:
either way I think however way we deal with highlights should be however way we deal with other actions that are in the same category.

if a highlight request is now being waited on, then colour overrides/inivisbile should be the same. Or at least, the unity utils should return a promise in similar manner even if you completely ignore it on the front end.

We have to consider that this is a third party facing API here as well, it'll be difficult to explain to customers why this one has a promise you can wait on to ensure the action has completed but the other 2 doesn't

which reminds me, can we also update the documentation comments accordingly please?

Reply:
@Amantini1997  so, update unity  util and the comment docs within the file to use promises and have a consistent api.